detox/detox.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
         android: {
             type: "android.emulator",
             device: {
-                avdName: "5.4_FWVGA_API_30"
+                avdName: "Pixel_3a_XL_API_30"
```

Comment:
What about a generic name instead?

Reply:
Changed to 'Pixel_API_30'. This is technically recognized as an ID of AVD in Android Studio.

Reply:
Yeap I know, but could be really generic.. like `EMULATOR_API_30`

Reply:
Thanks, changed to the suggested one 😅